src/Editor/Image/Panel.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { FC, useState } from 'react';
+import { CropperRef, Cropper } from 'react-advanced-cropper';
+import { EditorArgs } from '../EditorArgs';
+import { useWhppt } from '../../Context';
+import { WhpptInput } from '../../ui/components';
+import { WhpptGalleryTag } from '../../Gallery/GalleryTag';
+import { Gallery } from '../../Gallery';
+import { aspectRatios } from '../../Gallery/Model';
+import { DevicePicker } from './DevicePicker';
+import { ImageItemDataSize, PageImageItemData, ImageItemData } from './../../Gallery/Model/Image';
+import { ImageEditorOptions } from '../Editors';
```

Comment:
Because we use options?.sizes we might not need this import. We could use EditorOptions (Base type)